draft-ietf-quic-transport.md

```diff
@@ -288,16 +289,12 @@ processing.
 
 ## Connection Migration and Resilience to NAT Rebinding
 
-QUIC connections are identified by a Connection ID, a 64-bit unsigned number
-randomly generated by the server.  QUIC's consistent connection ID allows
-connections to survive changes to the client's IP and port, such as those caused
-by NAT rebindings or by the client changing network connectivity to a new
-address. QUIC provides automatic cryptographic verification of a rebound client,
-since the client continues to use the same session key for encrypting and
-decrypting packets. The consistent connection ID can be used to allow migration
-of the connection to a new server IP address as well, since the Connection ID
-remains consistent across changes in the client's and the server's network
-addresses.
+QUIC connections carry a connection ID, an opaque value assigned by the intended
+recipient of the packet.  Using a consistent connection ID allows connections to
+survive changes to the sender's IP and port, such as those caused by NAT
+rebindings or by changes in network connectivity.  QUIC provides mechanisms that
+verify addresses of endpoints that change network connectivity, while
+maintaining session continuity.
```

Comment:
This phrase doesn't feel right with a comma, but without the comma it's ambiguous which idea it's referring to.  Maybe reword?

Reply:
Difficult.  A complete redo maybe.

Reply:
"When an endpoint changes network connectivity, QUIC provides mechanisms to verify new addresses while maintaining session continuity.", maybe?